src/main/kotlin/calculator/StringAddCalculator.kt

```diff
@@ -10,14 +10,14 @@ object StringAddCalculator {
         StringDelimiterSeparator("^//(?<delimiter>.)\n(?<strings>.*)\$".toRegex())
     private val DEFAULT_DELIMITER: Regex = "[,:]".toRegex()
 
-    val String?.splitAndSum: Int
+    val String?.totalNumber: Int
```

Comment:
```suggestion
    val String?.totalNumber(): Int
```
제가 의도한 것은 "프로퍼티는 행동이 아닌 상태를 가져야 한다"는 것입니다.
totalNumber가 외부에서는 프로퍼티로 보이지만, getter의 구현부를 보면 `WholeNumber.Companion.totalNumber`에 접근을 하고 있습니다.
`WholeNumber.Companion.totalNumber`는 fold를 하고 있으므로 상태를 나타내기 보다는 행동을 나타내고 있습니다.
또한, 함수의 관점에서 본다면, getter는 두 가지 이상의 일을 하고 있습니다. (수식 분리, 합 연산 등)
프로퍼티는 상태만을 가질 수 있도록 전반적으로 개선해보면 어떨까요?

ref. [Item 16: Properties should represent state, not behavior](https://kt.academy/article/ek-property)